### DISCUSSION TITLE
Sites with dark or black logos

### DISCUSSION THREAD

**Discussion by hjpoe** [darkreader/darkreader#7649] [Q&A] [answered]
Some websites have logos that are already dark or black to begin with. Turning the background around them dark can make them difficult or impossible to see and read. 
Example: [bilio](https://www.biliomask.com/)
![image](https://user-images.githubusercontent.com/35583004/146128236-04f59785-6804-461a-a234-6323564fa91a.png)
Would it be possible to implement some way for these sites to have more visible and readable logos? A few ways of doing this might include:
- a lighter gray area around the navigation bar
- flip the logo to light color
- a lighter area around the logo itself, perhaps like a highlight underneath it

I can see one or other of these ways not working for some or all sites, so maybe a combination would be possible, or an entirely new way of doing it.
I'm sorry if this has been asked before, I looked in issues but didn't find anything. 
Please let me know if I can be of any help with this, I know a little CSS.

**Comment by Gusted** [comment, marked as answer]
Hi @hjpoe!

This is quite a common issue on sites with dark reader, luckily a easy fix is already available for such problems(which relates to your second point). Have a look [at the table that is listed under the section](https://github.com/darkreader/darkreader/blob/master/CONTRIBUTING.md#how-to-use-the-dev-tools), especially the "INVERT" one ;)

Regards,
Gusted

**Reply by hjpoe** [reply]
Thanks for the help :) 

I've submitted a pull request for this, and I may do a few more that I've found.

Many Thanks,
hjpoe
